cmd/cli/docker/docker_run_test.go

```diff
@@ -354,7 +354,7 @@ func (s *DockerRunSuite) TestTruncateReturn() {
 			ctx := context.Background()
 			_, out, err := s.ExecuteTestCobraCommand(
 				"docker", "run",
-				"ubuntu", "--", "perl", "-e", fmt.Sprintf(`print "=" x %d`, tc.inputLength),
+				"busybox:latest", "--", "perl", "-e", fmt.Sprintf(`print "=" x %d`, tc.inputLength),
```

Comment:
This test seems to be also expecting perl to be present, which is not on busybox

Reply:
Sorry I didn't complete the migration and should've marked the PR as draft. Fixed now